File: 171-Tree Height Measurement/171-Tree Height Measurement.go
func treeHeightMeasurement() {
    var numOfInput int
    fmt.Scanf("%d\n", &numOfInput)
    for i := 0; i < numOfInput; i++ {
        var d, b float64
        fmt.Scanf("%f %f\n", &d, &b)
        fmt.Print(int(math.Round(d*math.Tan((b-90.0)*math.Pi/180))), " ")
    }
}

File: 171-Tree Height Measurement/171-Tree Height Measurement.cs
 static public string task171(string Input)
        {
            string Output = "";
            Regex inputPtrn = new Regex(@"(\d+) (\d+,\d+)");
            double D = 0, A = 0;
            foreach (Match tree in inputPtrn.Matches(Input.Replace('.', ',')))
            {
                D = Convert.ToDouble(tree.Groups[1].Value);
                A = Convert.ToDouble(tree.Groups[2].Value) % 90;
                Output += Math.Round(D * Math.Tan(A * Math.PI / 180)) + " ";
            }
            return Output;
        }